README.md

```diff
@@ -1,29 +1,64 @@
+<p align="center">
+  <img src="https://avatars.githubusercontent.com/u/103479527" width="128px" height="128px" alt="Mica For Everyone, two blue squares logo">
+</p>
+<h1 align="center">Mica For Everyone!</h1>
+<p align="center">
+  <a target="_blank" href="https://github.com/MicaForEveryone/MicaForEveryone/releases">
+    <img alt="GitHub release (latest by date)" src="https://img.shields.io/github/v/release/MicaForEveryone/MicaForEveryone?color=%230A99D6&label=lastest%20version&style=for-the-badge">
+  </a>
+  <a style="text-decoration:none" href="https://github.com/MicaForEveryone/MicaForEveryone/releases" target="_blank">
+    <img alt="GitHub all releases" src="https://img.shields.io/github/downloads/MicaForEveryone/MicaForEveryone/total?color=%230A99D6&style=for-the-badge">
+  </a>
+</p>
 
-Jadi Mirmirani, an Iranian open-source activist and Mohsen Tahmasbi, an Iranian security researcher are among the arrested people in October 2022.
 
-# Mica For Everyone!
-Mica For Everyone is a tool to customize system backdrop on Win32 apps using [DwmSetWindowAttribute](https://docs.microsoft.com/en-us/windows/win32/api/dwmapi/nf-dwmapi-dwmsetwindowattribute) and other methods.
+**Mica For Everyone** is a tool to customize system backdrop on Win32 apps using [DwmSetWindowAttribute](https://docs.microsoft.com/en-us/windows/win32/api/dwmapi/nf-dwmapi-dwmsetwindowattribute) and other methods.
 It can apply Mica (or any other backdrop materials) on the non-client area (window frame) or background of supported apps and its behavior is customizable through a GUI and a config file.
 
-**NOTE**: Mica For Everyone is not responsible for rendering the effects you set, it just asks Windows to do that for you. If there's any problem with the effects it's a third-party issue. Try creating a rule for the affected apps and try different settings before opening an issue for it.
+> **NOTE**: Mica For Everyone is not responsible for rendering the effects you set, it just asks Windows to do that for you. If there's any problem with the effects it's a third-party issue. Try creating a rule for the affected apps and try different settings before opening an issue for it.
 
-## Config File
+## 🕹 How do I get it?
+Goto **Releases page > Assets > Click on MicaForEveryone-x64-Release-Installer.exe** or [just click here to download the latest version of MicaForEveryone.](https://github.com/MicaForEveryone/MicaForEveryone/releases)
+
+## 🛠 Config File
 For more information check our [wiki page](https://github.com/MicaForEveryone/MicaForEveryone/wiki/Config-File) and [default config file](MicaForEveryone/Resources/MicaForEveryone.conf).
 
-## Screenshots
+## 🖥 Screenshots
 
 ![Screenshot 1](Assets/1.png)
 
 ![Screenshot 2](Assets/2.png)
 
 ![Screenshot 3](Assets/3.png)
 
-## Frequency Asked Questions
+## 👋🏻 Samples
```

Comment:
I think it would be better without this.

Reply:
> I think it would be better without this.

done :)